### PR TITLE
[Feature]: Settings global storage

### PR DESCRIPTION
**Pull request by tux255**

### CODE REVIEW

File: src/components/ui/Checkboxes/Checkbox.jsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ function Checkbox({ checked, label, onChange }) {
       {label}
       <input
         onChange={handleChange}
+        defaultChecked={checked}
```
Comment (Nedilko) [inline src/components/ui/Checkboxes/Checkbox.jsx:22]: why do you need this?

File: src/components/ui/Toggles/ToggleSwitch.jsx [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+import { useState } from "react";
+import PropTypes from "prop-types";
 
 const bgLight = `bg-[url('./assets/light.svg')]`;
 const bgDark = `bg-[url('./assets/dark.svg')]`;
 
 ToggleSwitch.propTypes = {
   isOn: PropTypes.bool.isRequired,
   onChange: PropTypes.func.isRequired,
+  isDisabled: PropTypes.bool,
```
Comment (Nedilko) [inline src/components/ui/Toggles/ToggleSwitch.jsx:10]: better to use "positive" name convention - `isEnabled`

Comment (tux255) [reply src/components/ui/Toggles/ToggleSwitch.jsx:10]: checkbox does not show it's current value without it dunno why.

File: src/components/ui/Checkboxes/Checkbox.jsx (hunk 1 above)
Comment (tux255) [reply src/components/ui/Checkboxes/Checkbox.jsx:22]: that is right. I used value instead checked for checkbox :)

**Review by Nedilko — APPROVED** [review]

**Pull request merged**